lib/languageTag.js

```diff
@@ -0,0 +1,47 @@
+const { namedNode } = require('@rdfjs/data-model')
+
+const langString = namedNode('http://www.w3.org/1999/02/22-rdf-syntax-ns#langString')
+const xsdString = namedNode('http://www.w3.org/2001/XMLSchema#string')
```

Comment:
Not too happy about this as there are similar constructs elsewhere.

Reply:
Maybe create a `staticNamespaces.js` file which contains all the terms used only for comparing.

Reply:
So I looked a the `lib/namespace.js` module. Is it really necessary for it to be constructed with a specific factory. 

You will use the instance's factory there but in other code `@rdfjs/data-model` is used anyway so what's the point?

Reply:
The factory in `lib/namespace.js` is necessary when we create new nodes in the graph (e.g. `.addList`).

Reply:
So my point was that there are direct usages of `@rdfjs/data-model` but on the other hand `Clownface` does not import a factory and instead takes one as parameter.

And the RDF/XSD namespaces a little scattered